.github/workflows/airflow-operator-release-to-pypi.yml

```diff
@@ -9,14 +9,22 @@ jobs:
     strategy:
       matrix:
         go: [ '1.18' ]
+        python: [ '3.8', '3.9', '3.10' ]
+        include:
+          - tox-env: 'py38'
```

Comment:
What allows you to omit `python: '3.8'` here, while needing it for the other `tox-env` entries?

Reply:
This came from another PR that was just merged.  I'm mostly replicating it in the release action as our unit test build now has this.  

The reason we have these two is that tox requires py38 and the setup-action for python requires a version like '3.8'..  So..

Reply:
Oh I see your question.  I think maybe we include tox py38 as the default and the others are extra?